pkg/cvo/internal/operatorstatus.go

```diff
@@ -249,7 +249,7 @@ func waitForOperatorStatusToBeDone(ctx context.Context, interval time.Duration,
 		}
 
 		lastErr = &payload.UpdateError{
-			Nested:       nestedMessage,
+			Nested:       fmt.Errorf("cluster operator is available and not degraded but has not finished updating to target version"),
 			UpdateEffect: payload.UpdateEffectNone,
 			Reason:       "ClusterOperatorNotAvailable",
```

Comment:
I think `cluster operator is available...` and `ClusterOperatorNotAvailable` right next to each other is too big of a disconnect.  Can we bump the `Reason` to `ClusterOperatorUpdating` here to match?

Reply:
done